### DISCUSSION TITLE
Null object error when i load a character

### DISCUSSION THREAD

**Discussion by ValenG2002** [KadeArchive/Kade-Engine#1716] [Q&A]
hello, i need help, when I load the character I get the Null object reference error, can someone help me?
![image](https://user-images.githubusercontent.com/86801701/129155179-87565644-f634-41a4-ae45-ca54c87084b4.png)
![image](https://user-images.githubusercontent.com/86801701/129155233-4df21ce4-4027-4b09-ac19-33896f100b50.png)
![image](https://user-images.githubusercontent.com/86801701/129155370-c7cc026c-8b6b-492b-b369-19858e128289.png)
here is the code

**Comment by raf13lol** [comment]
what version of ke

(that is ke 1.6 and below)

**Reply by libracrumbs** [reply]
Same happening to me too, it happens sometimes in the main menu, and it happens while trying to offset the character (in this case Nonsense). I double-checked the code and it does not work. I use 1.6 because that is the only newest version that tutorials work on.

**Reply by ValenG2002** [reply]
The version is the 1.6

**Reply by Dancetrap** [reply]
I'm on the Updated 1.8 version and I don't know why it doesn't work
